src/config/types.ts

```diff
@@ -128,23 +128,16 @@ export interface ExpectOptsConfig {
     interval: number;
 }
 
-export interface MochaOpts {
-    /** milliseconds to wait before considering a test slow. */
-    slow?: number;
-
-    /** timeout in milliseconds or time string like '1s'. */
-    timeout?: number;
-
-    /** string or regexp to filter tests with. */
-    grep?: string | RegExp;
```

Comment:
All these options specified inside `Mocha.MochaOptions`, so I just use it. Moreover it has a lot of other options that user can use.

Reply:
ok